### PR TITLE
Change routes

### PR DESCRIPTION
**Pull request by JagerTheo**

### CODE REVIEW

**Comment by nibor51** [comment]
correction in progress

File: src/Controller/FightController.php [hunk 1]
```diff
@@ -36,6 +37,11 @@ public function initiate()
     }
     public function statusFight()
     {
+        // choose fighters
+        if (!isset($_SESSION['player1']) || !isset($_SESSION['player2'])) {
+            $fightersController = new FightersController();
+            return $fightersController->pick();
```
Comment (rocambille) [inline src/Controller/FightController.php:43]: pick may as well be a method of this controller. then you can go with:

    return $this->pick();

File: src/routes.php [hunk 2]
```diff
@@ -13,14 +13,14 @@
     'items/show' => ['ItemController', 'show', ['id']],
     'items/add' => ['ItemController', 'add',],
     'items/delete' => ['ItemController', 'delete',],
```
Comment (rocambille) [inline src/routes.php:15]: I guess now you can remove item stuff ;)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/Controller/FightController.php (hunk 1 above)
Comment (nibor51) [reply src/Controller/FightController.php:43]: FightController and FighterController not same files

File: src/routes.php (hunk 2 above)
Comment (nibor51) [reply src/routes.php]: Done

File: src/Controller/FightController.php (hunk 1 above)
Comment (rocambille) [reply src/Controller/FightController.php:43]: I didn't confirm my edition. the comment should be:

>pick may as well be in this controller. then you can go with:
>
>     return $this->pick();

Comment (nibor51) [reply src/Controller/FightController.php:43]: oh mb

Comment (nibor51) [reply src/Controller/FightController.php]: Done

**Pull request merged**